extension/extension/background.js

```diff
@@ -96,6 +97,10 @@ function websocket_logger(server) {
             event = JSON.stringify(event);
             queue.push(event);
         };
+        socket.onmessage = function(event) {
+            const message = event.data;
+            console.log('Received message:', message);
```

Comment:
Might be helpful to mark this as debug code, so we remember to remove once we get out of WiP stage.